cni-plugins/sriov-passthrough-cni/deploy/multus-cni.yaml

```diff
@@ -106,6 +106,7 @@ apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
   name: sriov-passthrough-cni
+  namespace: multus-cni-ns
 spec:
   config: |
     {
```

Comment:
maybe need also the change next line to 0.3.1 as the cni binary itself outputs 0.3.1